tests/test_horizon_scores.py

```
@@ -284,14 +286,36 @@ def test_uses_correct_headers(self, mock_post):
         assert headers["anthropic-version"] == "2023-06-01"
 
 
+class TestComputeGateThenRankScore:
+    """Tests for the deterministic gate-then-rank scoring formula."""
+
+    def test_action_class_adds_boosts(self):
+        assert compute_gate_then_rank_score("action", 70, True, True) == 90
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)